addOns/sstiscanner/src/main/java/org/zaproxy/zap/extension/sstiscanner/SSTIChallengeCallbackApi.java

```diff
@@ -0,0 +1,227 @@
+/*
+ * Zed Attack Proxy (ZAP) and its related class files.
+ *
+ * ZAP is an HTTP/HTTPS proxy for assessing web application security.
+ *
+ * Copyright 2019 The ZAP Development Team
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.zaproxy.zap.extension.sstiscanner;
+
+import java.security.SecureRandom;
+import java.util.*;
+import org.apache.commons.httpclient.URIException;
+import org.apache.log4j.Logger;
+import org.parosproxy.paros.control.Control;
+import org.parosproxy.paros.network.HttpMessage;
+import org.zaproxy.zap.extension.api.ApiException;
+import org.zaproxy.zap.extension.callback.CallbackImplementor;
+import org.zaproxy.zap.extension.callback.ExtensionCallback;
+
+/** @author DiogoMRSilva (2018) based on XXE plugin(yhawke) */
+public class SSTIChallengeCallbackApi implements CallbackImplementor {
+
+    private static final String PREFIX = "SSTI";
+    private static final int CHALLENGE_LENGTH = 6;
+
+    // The default expiration time for each callback (in millisecs)
+    private static final long CALLBACK_EXPIRE_TIME = 2 * 60 * 1000;
```

Comment:
This might be more clear as `TimeUnit.MINUTES.toMillis(2)`

Edit: There's likely other place in this PR that handle millis or time where leveraging TimeUnit would be advantageous.